scripts/pa11y.js

```diff
@@ -62,7 +61,7 @@ async function runPa11y() {
     ),
   );
 
-  urls.map((path) => {
+  urls.forEach((path) => {
```

Comment:
This stops eslint complaining that there is no return value as in this case we don't care about the return value